.github/workflows/main.yml

```diff
@@ -39,10 +39,10 @@ jobs:
       run: mkdir -p config/_default/; mkdir -p config/_default/; cat content/navigation.md | ./navigation2menu.js > config/_default/menu.en.json
 
     - name: Build WebChart HTML
-      run: hugo --config 'config-wc.toml' --baseURL "https://docs-qa.med-web.com/${{ steps.extract_branch.outputs.branch }}/wc/" --minify
+      run: hugo --config 'config-wc.toml' --baseURL "https://docs-qa.med-web.com/${{ steps.extract_branch.outputs.branch }}/wc/" --minify --environment staging
 
     - name: Build Enterprise Health HTML
-      run: hugo --config 'config-eh.toml' --baseURL "https://docs-qa.med-web.com/${{ steps.extract_branch.outputs.branch }}/eh/" --minify
+      run: hugo --config 'config-eh.toml' --baseURL "https://docs-qa.med-web.com/${{ steps.extract_branch.outputs.branch }}/eh/" --minify --environment staging
```

Comment:
do we want that change? shouldn't this stay production environment?

Reply:
I don't have access. I'm not sure if that even works.

Reply:
Can we add just this change:
`const level = (indentPart.length - 2)/4;`
for now, or do you also need the other ones right away? I just don’t want to mess up the configuration if we’re not sure we won’t accidentally destroy the entire Hugo-WikiDrive universe

Reply:
No, we do not need the rest